src/templates/promotions/discounts/index.html

```diff
@@ -26,7 +26,7 @@
 {% for discount in discounts %}
     {% set tableData = tableData|merge([{
         id: discount.id,
-        title: discount.name,
+        title: discount.name|t('site'),
```

Comment:
did you mean `|e` here?

Reply:
The title column data is escaped in the Vue app.

The use of `|t('site')` mimics the way Craft does it for things like field names as I assumed this was protocol?